### PR TITLE
Add intermediate solution to Algorithm-Roman-Numeral-Converter

### PR DESCRIPTION
**Pull request by mrthnmn**
A second (intermediate) solution has been added.

### CODE REVIEW

File: Algorithm-Roman-Numeral-Converter.md [hunk 1]
```diff
@@ -68,6 +68,70 @@ convertToRoman(36);
 - [For Loops](JS-For-Loop)
 - [While Loops](JS-while-Loop)
 
+## :intermediate: Intermediate Code Solution:
```
Comment (atjonathan) [inline Algorithm-Roman-Numeral-Converter.md]: ~~`:intermediate:`~~ `:sunflower:`

File: Algorithm-Roman-Numeral-Converter.md [hunk 2]
```diff
@@ -68,6 +68,70 @@ convertToRoman(36);
...
+  return romanNumber.reverse().join("").toString();
+}
+
+// test here
+convertToRoman(97);
+```
+
+:rocket: [Run Code](https://repl.it/C1YV)
+
+## Code Explanation:
```
Comment (atjonathan) [inline Algorithm-Roman-Numeral-Converter.md]: ~~`## Code Explanation:`~~ `### Code Explanation:`

**Comment by Rafase282** [comment]
Please add a section to our articles related to your solution.

**Comment by mrthnmn** [comment]
I'm afraid I'm not sure what you want. I thought I was editing the existing
wiki article on the Roman Numeral Converter Algorithm challenge. Do you
want a more-thorough explanation of the code I added or a completely new
wiki article?

**Comment by Rafase282** [comment]
Please take a look here https://github.com/FreeCodeCamp/wiki/issues/1094

We have a template for solutions, and you adding a new one also requires a section with related links. I was asking to add it and use links form our wiki if possible. Sorry for not being clear before.

File: Algorithm-Roman-Numeral-Converter.md [hunk 3]
```diff
@@ -6,6 +6,13 @@
 
 You will create a program that converts an integer to a Roman Numeral.
 
+#### Relevant Links
+
+- [Roman Numerals](http://www.mathsisfun.com/roman-numerals.html)
+- [Array.splice()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice)
+- Array.indexOf()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf)
```
Comment (atjonathan) [inline Algorithm-Roman-Numeral-Converter.md]: ~~`Array.indexOf()]`~~ `[Array.indexOf()]`

File: Algorithm-Roman-Numeral-Converter.md [hunk 4]
```diff
@@ -65,14 +72,83 @@ convertToRoman(36);
 
 #### Relevant Links
 
-- [For Loops](JS-For-Loop)
-- [While Loops](JS-while-Loop)
+- [For Loops](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for)
```
Comment (atjonathan) [inline Algorithm-Roman-Numeral-Converter.md]: Change this so it links to our internal article: [`JS-For-Loop`](https://github.com/FreeCodeCamp/wiki/blob/master/JS-For-Loop.md).

File: Algorithm-Roman-Numeral-Converter.md [hunk 5]
```diff
@@ -65,14 +72,83 @@ convertToRoman(36);
...
+- [While Loops](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/while)
```
Comment (atjonathan) [inline Algorithm-Roman-Numeral-Converter.md]: Change this so it links to our internal article: [`JS-While-Loop`](https://github.com/FreeCodeCamp/wiki/blob/master/JS-While-Loop.md).

File: Algorithm-Roman-Numeral-Converter.md [hunk 6]
```diff
@@ -6,6 +6,13 @@
 
 You will create a program that converts an integer to a Roman Numeral.
 
+#### Relevant Links
+
+- [Roman Numerals](http://www.mathsisfun.com/roman-numerals.html)
+- [Array.splice()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice)
```
Comment (atjonathan) [inline Algorithm-Roman-Numeral-Converter.md]: Change this so it links to our internal article: [`JS-Array-Prototype-Splice`](https://github.com/FreeCodeCamp/wiki/blob/master/JS-Array-Prototype-Splice.md).

File: Algorithm-Roman-Numeral-Converter.md [hunk 7]
```diff
@@ -6,6 +6,13 @@
...
+- Array.indexOf()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf)
+- [Array.join()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join)
```
Comment (atjonathan) [inline Algorithm-Roman-Numeral-Converter.md]: Change this so it links to our internal article: [`JS-Array-Prototype-Join`](https://github.com/FreeCodeCamp/wiki/blob/master/JS-Array-Prototype-Join.md).

File: Algorithm-Roman-Numeral-Converter.md [hunk 8]
```diff
@@ -65,14 +72,83 @@ convertToRoman(36);
...
+
+- Create an array of Roman Numerals (`romans`).
+- Use a for loop to create an array of the digits (`ints`) in the number.
+- Loop through the array of digits (base 10) and as you do, increment the Roman Numeral (base 5) index by 2 (`numeral = romans[i*2]`).
+- Within the loop, use Switch Case to push the proper Roman Numerals (backwards) onto that array.
+- Reverse the Roman Numerals array and turn it into a string.
+
+#### Relevant Links
+
+- [For Loops](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for)
```
Comment (atjonathan) [inline Algorithm-Roman-Numeral-Converter.md]: Change this so it links to our internal article: [`JS-For-Loop`](https://github.com/FreeCodeCamp/wiki/blob/master/JS-For-Loop.md).

File: Algorithm-Roman-Numeral-Converter.md [hunk 9]
```diff
@@ -65,14 +72,83 @@ convertToRoman(36);
...
+- [While Loops](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/while)
```
Comment (atjonathan) [inline Algorithm-Roman-Numeral-Converter.md]: Change this so it links to our internal article: [`JS-While-Loop`](https://github.com/FreeCodeCamp/wiki/blob/master/JS-While-Loop.md).

**Comment by Rafase282** [comment]
Please try to fix the conflicts on your branch. Feel free to come to the wiki chat if you need help.

File: Algorithm-Roman-Numeral-Converter.md [hunk 10]
```diff
@@ -6,6 +6,13 @@
 
 You will create a program that converts an integer to a Roman Numeral.
 
+#### Relevant Links
+
+- [Roman Numerals](http://www.mathsisfun.com/roman-numerals.html)
+- [Array.splice()](https://github.com/FreeCodeCamp/wiki/blob/master/JS-Array-Prototype-Splice.md)
+- Array.indexOf()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf)
```
Comment (Rafase282) [inline Algorithm-Roman-Numeral-Converter.md]:
```diff
-- Array.indexOf()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf)
+- [Array.indexOf()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf)
```

File: Algorithm-Roman-Numeral-Converter.md [hunk 11]
```diff
@@ -6,6 +6,13 @@
 
 You will create a program that converts an integer to a Roman Numeral.
 
+#### Relevant Links
+
+- [Roman Numerals](http://www.mathsisfun.com/roman-numerals.html)
+- [Array.splice()](https://github.com/FreeCodeCamp/wiki/blob/master/JS-Array-Prototype-Splice.md)
```
Comment (Rafase282) [inline Algorithm-Roman-Numeral-Converter.md]:
```diff
-- [Array.splice()](https://github.com/FreeCodeCamp/wiki/blob/master/JS-Array-Prototype-Splice.md)
+- [Array.splice()](JS-Array-Prototype-Splice)
```

File: Algorithm-Roman-Numeral-Converter.md [hunk 12]
```diff
@@ -6,6 +6,13 @@
...
+- Array.indexOf()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf)
+- [Array.join()](https://github.com/FreeCodeCamp/wiki/blob/master/JS-Array-Prototype-Join.md)
```
Comment (Rafase282) [inline Algorithm-Roman-Numeral-Converter.md]:
```diff
-- [Array.join()](https://github.com/FreeCodeCamp/wiki/blob/master/JS-Array-Prototype-Join.md)
+- [Array.join()](JS-Array-Prototype-Join)
```

File: Algorithm-Roman-Numeral-Converter.md [hunk 13]
```diff
@@ -65,14 +72,83 @@ convertToRoman(36);
 
 #### Relevant Links
 
-- [For Loops](JS-For-Loop)
-- [While Loops](JS-while-Loop)
+- [For Loops](https://github.com/FreeCodeCamp/wiki/blob/master/JS-For-Loop.md)
```
Comment (Rafase282) [inline Algorithm-Roman-Numeral-Converter.md]:
```diff
-- [For Loops](https://github.com/FreeCodeCamp/wiki/blob/master/JS-For-Loop.md)
+- [For Loops](JS-For-Loop)
```

File: Algorithm-Roman-Numeral-Converter.md [hunk 14]
```diff
@@ -65,14 +72,83 @@ convertToRoman(36);
...
+- [While Loops](https://github.com/FreeCodeCamp/wiki/blob/master/JS-While-Loop.md)
```
Comment (Rafase282) [inline Algorithm-Roman-Numeral-Converter.md]:
```diff
-- [While Loops](https://github.com/FreeCodeCamp/wiki/blob/master/JS-While-Loop.md)
+- [While Loops](JS-While-Loop)
```

File: Algorithm-Roman-Numeral-Converter.md [hunk 15]
```diff
@@ -65,14 +72,83 @@ convertToRoman(36);
...
+
+- Create an array of Roman Numerals (`romans`).
+- Use a for loop to create an array of the digits (`ints`) in the number.
+- Loop through the array of digits (base 10) and as you do, increment the Roman Numeral (base 5) index by 2 (`numeral = romans[i*2]`).
+- Within the loop, use Switch Case to push the proper Roman Numerals (backwards) onto that array.
+- Reverse the Roman Numerals array and turn it into a string.
+
+#### Relevant Links
+
+- [For Loops](https://github.com/FreeCodeCamp/wiki/blob/master/JS-For-Loop.md)
```
Comment (Rafase282) [inline Algorithm-Roman-Numeral-Converter.md]:
```diff
-- [For Loops](https://github.com/FreeCodeCamp/wiki/blob/master/JS-For-Loop.md)
+- [For Loops](JS-For-Loop)
```

File: Algorithm-Roman-Numeral-Converter.md [hunk 16]
```diff
@@ -65,14 +72,83 @@ convertToRoman(36);
...
+- [While Loops](https://github.com/FreeCodeCamp/wiki/blob/master/JS-While-Loop.md)
```
Comment (Rafase282) [inline Algorithm-Roman-Numeral-Converter.md]:
```diff
-- [While Loops](https://github.com/FreeCodeCamp/wiki/blob/master/JS-While-Loop.md)
+- [While Loops](JS-While-Loop)
```

**Comment by atjonathan** [comment]
@mrthnmn, please resolve merge conflicts and implement suggestions :wink:

**Comment by mrthnmn** [comment]
I will do on Monday. I wasn't able to on Saturday because I was behind a
corporate proxy.

**Comment by alayek** [comment]
@mrthnmn this branch has conflicts. You might need to rebase the branch.

**Comment by mrthnmn** [comment]
Attempt to resolve conflicts

**Pull request closed without merging**